Dockerfile

```diff
@@ -7,32 +11,78 @@ LABEL \
     org.label-schema.vcs-url="https://github.com/OCR-D/ocrd_tesserocr" \
     org.label-schema.build-date=$BUILD_DATE
 
-ENV DEBIAN_FRONTEND noninteractive
 ENV PYTHONIOENCODING utf8
 
+# set TESSDATA_PREFIX
+ENV TESSDATA_PREFIX /usr/local/share/tessdata
+
+# set frontend non-interactive to silence interactive tzdata config
+ARG DEBIAN_FRONTEND=noninteractive
+
+
+# install common tools and tesseract build dependencies
+# use provided leptonica
+# tzdata required for proper timezone settings
+RUN apt-get update && apt-get install -y \
+	apt-utils \
+	build-essential \
+	g++ \
+	git \
+	libjpeg-dev \
+	libgif-dev \
+	libwebp-dev \
+	libopenjp2-7-dev \
+	libpng-dev \
+	libtiff-dev \
+	libtool \
+	pkg-config \
+	tzdata \
+	xzgv \
+	zlib1g-dev \
+	libleptonica-dev \
+	libpango1.0-dev \
+	libicu-dev \
+	autotools-dev \
+	automake \
+	libcurl4-nss-dev \
+	libarchive-dev
```

Comment:
Could you please move these to the `deps-ubuntu` target (and remove the PPA stuff there)?

Reply:
thx!